roles/openshift_daemonset_config/templates/daemonset.yml.j2

```diff
@@ -0,0 +1,142 @@
+---
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  name: {{ openshift_daemonset_config_daemonset_name }}
+  annotations:
+    kubernetes.io/description: |
+      This daemon set manages the operational configuration for a cluster and ensures all nodes have
+      a concrete set of config in place. It could also use a local ansible run against the /host directory.
+spec:
+  selector:
+    matchLabels:
+      app: {{ openshift_daemonset_config_daemonset_name }}
+      confighosts: ops
+      ops.openshift.io/role: operations
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: {{ openshift_daemonset_config_daemonset_name }}
+        confighosts: ops
+        ops.openshift.io/role: operations
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+{% if openshift_daemonset_config_node_selector is defined and openshift_daemonset_config_node_selector != {} %}
+      nodeSelector: {{ openshift_daemonset_config_node_selector | to_json }}
+{% endif %}
+      serviceAccountName: {{ openshift_daemonset_config_sa_name }}
+      hostNetwork: true
+      hostPID: true
+      hostIPC: true
+      containers:
+      - name: config
+        image: centos:7
```

Comment:
In the future we'll probably want to use an existing image we're sure they cluster already has access to like openshift/origin-pod or openshift3/ose-pod based on whether or not it's origin/ocp. Adding another image here means that disconnected users will need to ensure another image is sync'd to use this functionality.

Reply:
@sdodson, I can make that into a variable and default it to something and someone can overwrite the image if they want.

Reply:
@sdodson I can use one of the ose pods as well.  @sdodson @smarterclayton, any suggestions for this?

Reply:
I think the pod image is fine, it's a full rhel/centos image and it's going to need to be on all hosts anyway just to run pods. Just need to make sure you're overriding the entrypoint. But clayton may have different opinion.